test/Microsoft.DotNet.Cli.Utils.Tests/GivenAProjectDependenciesCommandFactory.cs

```diff
@@ -147,5 +147,36 @@ public void It_resolves_desktop_apps_using_configuration_passed_to_create()
             command.CommandName.Should().Contain(Path.Combine(testInstance.TestRoot, "bin", configuration));
             Path.GetFileName(command.CommandName).Should().Be("dotnet-desktop-and-portable.exe");
         }
+
+        [WindowsOnlyFact]
```

Comment:
Good catch. One of the reasons I used [Path.Combine](https://github.com/dotnet/cli/pull/2620/files/3c4c5753ce3e7e1a390caa3d85b07278e6a2eca8#diff-bda8800dd549b6d314e4756ecc4a26a0R179) was to make it run on all platforms.